src/raftstore/store/region_snapshot.rs

```diff
@@ -539,13 +539,19 @@ mod tests {
             iter.reverse_seek(&Key::from_encoded(b"a7".to_vec()), &mut statistics)
                 .unwrap()
         );
-        let mut pair = (iter.key().to_vec(), iter.value().to_vec());
+        let mut pair = (
+            iter.key(&mut statistics).to_vec(),
```

Comment:
em, can you extract them to a function?

Reply:
don't worry. many of them are removed in Mvcc refinement PR later.